### DISCUSSION TITLE
Asumir Trade válido

### DISCUSSION THREAD

**Discussion by blampasona18** [IIC2133-PUC/2024-2#44] [Tarea 0] [answered]
Hola! Para el evento TRADE-PUFFLE, podemos asumir que este siempre será válido? Es decir que los puffles correspondientes estén con el penguin asociado. Pregunto porque en los otros eventos sale explícitamente que se pueden asumir como válidos, pero en este no.
Gracias

**Comment by tomastrivino** [comment, marked as answer]
Hola,
Según enunciado 

`Al entregar IDs en los eventos se puede asumir que estas siempre serán válidas.`

Con lo que puedes asumir que se entregarán IDs válidos.
